alby/alby_oauth_service.go

```diff
@@ -839,15 +839,9 @@ func (svc *albyOAuthService) activateAlbyAccountNWCNode(ctx context.Context) err
 
 func (svc *albyOAuthService) GetChannelPeerSuggestions(ctx context.Context) ([]ChannelPeerSuggestion, error) {
 
-	token, err := svc.fetchUserToken(ctx)
-	if err != nil {
-		logger.Logger.WithError(err).Error("Failed to fetch user token")
-		return nil, err
-	}
-
-	client := svc.oauthConf.Client(ctx, token)
+	client := &http.Client{Timeout: 10 * time.Second}
 
-	req, err := http.NewRequest("GET", fmt.Sprintf("%s/internal/channel_suggestions", svc.cfg.GetEnv().AlbyAPIURL), nil)
+	req, err := http.NewRequest("GET", "https://getalby.com/api/internal/channel_suggestions", nil)
```

Comment:
Why remove the host being loaded from the env variable here? :thinking:

Reply:
I've moved these to constants in this file instead of being in the config, and switched this back to using a constant,